src/pages/ProductPage.jsx

```diff
@@ -6,7 +6,7 @@ import hostImage from "../assets/images/host-raus.webp";
 import MapView from "../components/MapView/MapView";
 import mapViewSampleImg from "./../assets/map-view-sample.png";
 import ProductGallery from "../components/ProductGallery/ProductGallery";
-import PersonProfile from "../components/PersonProfile/PersonProfile";
+import PersonProfile from "../components/MeetYourHostLeft/MeetYourHostLeft";
```

Comment:
As you have transferred this component to another place, it is better to remove PersonProfile from here.